### PR TITLE
Support google-java-format up to version 1.11.0 

### PR DESCRIPTION
**Pull request by aecio**
This PR updates the code to run the tests and CI on all recent versions of `google-java-format`. Builds on PR #64 (might need to merge that one first).

The `guava` dependency has been updated to 30.0.0 since the recent versions of `google-java-format` use APIs that are only avaiable on that version and newer.

CI is green: https://app.travis-ci.com/github/aecio/google-java-format-gradle-plugin/builds/238966456

Thanks for this great plugin!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 build/
 /.gradle/
+.idea/
```
Comment (sherter) [inline .gitignore:3]: See comment in other PR #64

**Review by sherter — CHANGES_REQUESTED** [review]
See the comment in the other PR. Otherwise looks good. Thanks!

File: .gitignore (hunk 1 above)
Comment (aecio) [reply .gitignore]: Fixed.

**Comment by sherter** [comment]
thanks!

**Pull request merged**